zebra/zebra_pbr.c

```diff
@@ -600,8 +606,12 @@ void kernel_pbr_iptable_add_del_status(struct zebra_pbr_iptable *iptable,
 		zsend_iptable_notify_owner(iptable, ZAPI_IPTABLE_FAIL_INSTALL);
 		break;
 	case SOUTHBOUND_DELETE_SUCCESS:
+		zsend_iptable_notify_owner(iptable,
+					   ZAPI_IPTABLE_REMOVED);
+		break;
 	case SOUTHBOUND_DELETE_FAILURE:
-		/* TODO : handling of delete event */
+		zsend_iptable_notify_owner(iptable,
+					   ZAPI_IPTABLE_REMOVED);
```

Comment:
Currently, the return status of delete is ignored by upper layer ( BGP) like it was initially with PBRd and unlike it is not for PBRd.
I would like to handle the removal return status handling on BGP, separately, because it requires more changes. If you agree, what I could do is remove the remove notification.
ok?

Reply:
I changed this to REMOVE_FAILURE, as well for IP RULE too